lib/engine/game/g_2038/meta.rb

```diff
@@ -16,6 +16,14 @@ module Meta
         GAME_RULES_URL = 'https://boardgamegeek.com/filepage/135017/2038-english-rules-and-supplements'
 
         PLAYER_RANGE = [3, 6].freeze
+
+        OPTIONAL_RULES = [
+          {
+            sym: :optional_short_game,
+            short_name: 'Short Game',
+            desc: 'Play a shorter game',
```

Comment:
Could the description be more specific? In the rules file I found it says "Remove $6,000 and 2 Phase II ships from the game." Could that text be used? Or if we want to paraphrase instead, "Play with a $4,000 bank and only two 4dc3 ships/trains" or something like that?

Reply:
Just found the spot with the full Short Game summary, maybe the description should just say to see the bottom right of page 3 for the short game rules?

Reply:
Sure, I didn't do much with it yet because the only thing even considered at the moment is the par'ing of TSI.  If you need the description fixed first I can do that.